tests/ut/azure_iot_hub_client_ut.c

```diff
@@ -495,7 +495,10 @@ static void testAzureIoTHubClient_SendTelemetry_InvalidArgFailure( void ** ppvSt
     /* Fail if the hub client is NULL. */
     assert_int_equal( AzureIoTHubClient_SendTelemetry( NULL,
                                                        ucTestTelemetryPayload,
-                                                       sizeof( ucTestTelemetryPayload ) - 1, NULL ),
+                                                       sizeof( ucTestTelemetryPayload ) - 1,
+                                                       NULL,
+                                                       eAzureIoTHubMessageQoS0,
```

Comment:
>eAzureIoTHubMessageQoS0 [](http://example.com/codeflow?start=55&length=23)

please add QoS1 test cases also. In that test you can be creative to verify that MQTT layer is correctly getting the QoS from middleware layer.